plugins/ComposeAuth/src/jsMain/kotlin/io/github/jan/supabase/compose/auth/composable/AppleAuth.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.jan.supabase.compose.auth.composable
+
+import androidx.compose.runtime.Composable
+import io.github.jan.supabase.compose.auth.ComposeAuth
+
+
```

Comment:
I should probably not mark every small indentation problem, just check that you only have one new line as indentation